mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -20,8 +32,27 @@ AUI.add(
 					initializer: function(config) {
 						var instance = this;
 
+						new A.CharCounter(
+							{
+								counter: '.counter',
+								input: '.input-item',
+								maxLength: 35
+							});
+
 						var todoList = instance.one('.task-container ul');
 
+						var finishedTask = A.one('.finished-task');
+
+						var remainingDiv = instance.one('.remaining-tasks-count');
+
+						remainingDiv.html('1');
+
+						var taskHistory = instance.one('.task-history table');
+
+						taskHistory.hide();
```

Comment:
Why are we hiding the element on first dom query? It should be hidden by default, either through css or ```hide``` class. This way the element will blink on page load.